### PR TITLE
feat: add more telemetry for statement and keyviz

### PR DESCRIPTION
**Pull request by baurine**
refine telemetry and bump `tidb-dashboard-for-dbaas` version as well.

### CODE REVIEW

File: ui/packages/tidb-dashboard-for-dbaas/public/test-iframe/parent.html [hunk 1]
```diff
@@ -47,7 +47,7 @@
         <a href="#/slow_query">SlowQuery</a>
         <a href="#/keyviz">KeyViz</a>
         <a href="#/topsql">TopSQL</a>
-        <a href="#/overview">Metrics</a>
+        <a href="#/metrics">Metrics</a>
```
Comment (YiniXu9506) [inline ui/packages/tidb-dashboard-for-dbaas/public/test-iframe/parent.html:50]:
```suggestion
        <a href="#/monitoring">Monitoring</a>
```

The name `Metrics` had been changed to `Monitoring` in this [PR](https://github.com/pingcap/tidb-dashboard/pull/1370), would you please update this?

Comment (baurine) [reply ui/packages/tidb-dashboard-for-dbaas/public/test-iframe/parent.html:50]: I will update it after I publish the new tidb-dashboard-for-dbaas, in published tidb-dashboard-for-dbaas 0.0.15, it is still `/metrics`.

Comment (baurine) [reply ui/packages/tidb-dashboard-for-dbaas/public/test-iframe/parent.html:50]: Let me publish tidb-dashboard-for-dbaas 0.0.16

Comment (YiniXu9506) [reply ui/packages/tidb-dashboard-for-dbaas/public/test-iframe/parent.html:50]: Got it!

Comment (baurine) [reply ui/packages/tidb-dashboard-for-dbaas/public/test-iframe/parent.html]: image

done!

**Review by YiniXu9506 — APPROVED** [review]
Reset LGTM

**Review by shhdgit — APPROVED** [review]

**Pull request merged**